docs/reference/buildx_rm.md

```diff
@@ -1,15 +1,11 @@
 # buildx rm
 
 ```
-Usage:  docker buildx rm [NAME]
+docker buildx rm [NAME]
+```
 
+<!---MARKER_GEN_START-->
 Remove a builder instance
 
-Options:
-      --builder string   Override the configured builder instance
-```
-
-## Description
 
-Removes the specified or current builder. It is a no-op attempting to remove the
-default builder.
+<!---MARKER_GEN_END-->
```

Comment:
I guess this was not removed on purpose (I can update if it was indeed accidental)


```suggestion
<!---MARKER_GEN_END-->

## Description

 Removes the specified or current builder. It is a no-op attempting to remove the
 default builder.
```

Reply:
I think this was accidental indeed. I removed something where there was no actual description and the generated line from the command description was just repeated. But this one does have a little bit more context.

Reply:
Fixed